test/ILLink.RoslynAnalyzer.Tests/AvoidAssemblyLocationInSingleFileTests.cs

```diff
@@ -19,6 +19,33 @@ static Task VerifySingleFileAnalyzer (string source, params DiagnosticResult[] e
 				expected);
 		}
 
+		// TODO: Remove this once we have the new attribute available for linker testing.
+		static Task VerifySingleFileAnalyzerWithRequiresAssemblyFilesAttribute (string source, params DiagnosticResult[] expected)
+		{
+			source = @"
+using System.Reflection;
```

Comment:
@vitek-karas I thought that we had the attribute already available since we already bump the linker to net6.0, but if I delete this block of code Roslyn will complain about System.Diagnostics.CodeAnalysis not having RequiresAssemblyFilesAttribute. I wasn't sure if I was missing anything 😵